plugins/woocommerce/changelog/44633-fix-available-express-payment-method-removal

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: fix
+
+Fix the removal of the express payment method from the state.
```

Comment:
This could be improved, it doesn't really describe the fix very well.

Perhaps: "Fixed a bug where express payment methods could not be removed from the `wc/store/payment` data store"